draft-normington-jsonpath-latest.xml

```diff
@@ -218,6 +225,94 @@ dot-child-name = TBD
                   </t>
                </section>       
             </section>       
+            <section numbered="true" toc="default">
+               <name>Union Matcher</name>
+               <section numbered="true" toc="default">
+                  <name>Syntax</name>
+                  <t>A union matcher consists of one or more union elements.
+                  </t>
+                  <sourcecode type="abnf">
+matcher =/ union
+union = %x5B ws union-elements ws %x5D ; [...]
+ws = *%x20                             ; zero or more spaces
+union-elements = union-element /
+                 union-element ws %x2C ws union-elements
+                                       ; ,-separated list
+                  </sourcecode>
+               </section>       
+               <section numbered="true" toc="default">
+                  <name>Semantics</name>
+                  <t>A union matches any node which is matched by at least one of the union elements and selects the concatenation of the
```

Comment:
> by at least one of the union

It is a question if union should "merge" union results or "concatenate" the results. We [have a consensus](https://cburgmer.github.io/json-path-comparison/results/union_with_numbers_in_decreasing_order.html) for reverse ordered indexes, that shows us that we should define the order of merge/concatenation; but the situation of fetching the same index twice (like `$[1,1]`) is still not investigated.

But this question can be discussed later.

Reply:
@remorhaz I suggest you raise an issue on this so we don't forget it.